cloudflare/workers/app.ts

```diff
@@ -1,20 +1,15 @@
 import { createRequestHandler } from "react-router";
 
-declare global {
-  interface CloudflareEnvironment extends Env {}
-}
-
```

Comment:
This is not necessary as the same type can be accessed globally via `Cloudflare.Env` or `Env`.